### PR TITLE
chore(emailfwd): change senior email forward message

### PR DESCRIPTION
**Pull request by shahsalonik**
## Proposed changes
- Changes "By the end of the summer after graduation" to "At the conclusion of the school year"

## Brief description of rationale
Makes more sense for seniors (because they won't be coming back to TJ)

### CODE REVIEW

File: intranet/templates/emailfwd/senior_forward.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h2>
         Senior Email Forwarding
     </h2>
 
-    <p><b>By the end of the summer after graduation, your TJ Network Account will be deleted.</b> However, you can configure messages sent to your TJ Email address, <b>{{ request.user.tj_email }}</b>, to forward to another address (such as your university email or a personal email). You will not be able to log into Webmail. Once set at the conclusion of the summer, this address can't be changed.</p>
+    <p><b>At the conclusion of the school year, your TJ Network Account will be deleted.</b> However, you can configure messages sent to your TJ Email address, <b>{{ request.user.tj_email }}</b>, to forward to another address (such as your university email or a personal email). You will not be able to log into Webmail. Once set at the conclusion of the summer, this address can't be changed.</p>
```
Comment (etnguyen03) [inline intranet/templates/emailfwd/senior_forward.html:33]:
```suggestion
    <p><b>At the conclusion of the school year, your TJ Network Account will be deleted.</b> However, you can configure messages sent to your TJ Email address, <b>{{ request.user.tj_email }}</b>, to forward to another address (such as your university email or a personal email). You will not be able to log into Webmail. Once set at the conclusion of the school year, this address can't be changed.</p>
```

**Review by etnguyen03 — CHANGES_REQUESTED** [review]
Also, please amend your commit message to follow the [Conventional Commits standard](https://www.conventionalcommits.org/en/v1.0.0/) -- using your PR title is fine.

File: intranet/templates/emailfwd/senior_forward.html (hunk 1 above)
Comment (shahsalonik) [reply intranet/templates/emailfwd/senior_forward.html]: Hi! I just changed it using your recommendations. Please let me know if I need to change anything else.

**Review by etnguyen03 — CHANGES_REQUESTED** [review]
Please:

* change `changed` to `change` - commit messages, in general but especially on Ion, use the imperative tense.
* squash your commits.

Otherwise, LGTM

**Comment by etnguyen03** [comment]
Superseded by #1145

**Pull request closed without merging**